labs-ios-starter/Model/Apollo/Web Auth/Member.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Member.swift
+//  labs-ios-starter
+//
+//  Created by Kenny on 9/8/20.
+//  Copyright © 2020 Lambda, Inc. All rights reserved.
+//
+
+import Foundation
+
+
+/// Standard user, leader if topic originator
+struct Member: Equatable, Codable {
```

Comment:
could also call this User

Reply:
I think `User` is a standard Swift type in iOS14 which they may want to use. Doing this to avoid a naming collision - but good catch. In this case User would actually be more appropriate/semantic since it's a catch-all